src/backend/gporca/data/dxl/minidump/DeleteMismatchedDistribution.mdp

```diff
@@ -506,14 +496,28 @@
                       <dxl:Ident ColId="9" ColName="gp_segment_id" TypeMdid="0.23.1.0"/>
                     </dxl:ProjElem>
                   </dxl:ProjList>
-                  <dxl:Filter/>
-                  <dxl:SortingColumnList/>
-                  <dxl:HashExprList>
-                    <dxl:HashExpr>
-                      <dxl:Ident ColId="0" ColName="i" TypeMdid="0.23.1.0"/>
-                    </dxl:HashExpr>
-                  </dxl:HashExprList>
-                  <dxl:Sequence>
+                  <dxl:PartitionSelector RelationMdid="0.24803.1.0" PartitionLevels="1" ScanId="1">
+                    <dxl:Properties>
+                      <dxl:Cost StartupCost="10" TotalCost="100" Rows="100" Width="4"/>
+                    </dxl:Properties>
+                    <dxl:ProjList/>
+                    <dxl:PartEqFilters>
+                      <dxl:ConstValue TypeMdid="0.16.1.0" Value="true"/>
+                    </dxl:PartEqFilters>
+                    <dxl:PartFilters>
+                      <dxl:ConstValue TypeMdid="0.16.1.0" Value="true"/>
+                    </dxl:PartFilters>
+                    <dxl:ResidualFilter>
+                      <dxl:ConstValue TypeMdid="0.16.1.0" Value="true"/>
+                    </dxl:ResidualFilter>
+                    <dxl:PropagationExpression>
+                      <dxl:ConstValue TypeMdid="0.23.1.0" Value="1"/>
+                    </dxl:PropagationExpression>
+                    <dxl:PrintableFilter>
+                      <dxl:ConstValue TypeMdid="0.16.1.0" Value="true"/>
+                    </dxl:PrintableFilter>
+                  </dxl:PartitionSelector>
```

Comment:
It seems that change in this test case doesn't regard to our fix. @dnskvlnk could you provide some arguments that this change makes sense and correct?

Reply:
Here we remove `PartitionSelector` node between `Result` and `RoutedDistributedMotion` nodes from query plan:
transform from
```
     Delete  (cost=0.00..862.03 rows=1 width=1)
       ->  Result  (cost=0.00..862.00 rows=1 width=26)
             ->  Partition Selector for pt2  (cost=0.00..862.00 rows=1 width=22)
                   ->  Explicit Redistribute Motion 4:4  (slice2; segments: 4)  (cost=0.00..862.00 rows=1 width=22)
                   ...
```
to
```
     Delete  (cost=0.00..862.03 rows=1 width=1)
       ->  Result  (cost=0.00..862.00 rows=1 width=26)
               ->  Explicit Redistribute Motion 4:4  (slice2; segments: 4)  (cost=0.00..862.00 rows=1 width=22)
               ...
```
as it doesn't add column with partition oid value.